lib/segment/src/vector_storage/mod.rs

```diff
@@ -24,21 +24,3 @@ mod query_scorer;
 
 pub use raw_scorer::*;
 pub use vector_storage_base::*;
-
-/// Calculates the quotient of self and rhs, rounding the result towards positive infinity.
-///
-/// # Panics
-///
-/// This function will panic if rhs is zero.
-///
-/// # Overflow behavior
-///
-/// On overflow, this function will panic if overflow checks are enabled (default in debug mode)
-/// and wrap if overflow checks are disabled (default in release mode).
-// Can be replaced with stdlib once it stabilizes:
-// - <https://github.com/rust-lang/rust/issues/88581>
-// - <https://doc.rust-lang.org/std/primitive.usize.html#method.div_ceil>
-#[inline]
-pub(crate) const fn div_ceil(a: usize, b: usize) -> usize {
-    (a + b - 1) / b
```

Comment:
Is this a copy paste from the standard library?

Reply:
Basically, yes. We needed this because it wasn't stabilized yet. Now it can be removed as what is provided in the stdlib is a full replacement.

Reply:
I wanted to double check the std implementation but the source only contains a macro `uint_impl!`

Have you been able to validate it?